bamboo/bootstrap-integration-tests.sh

```diff
@@ -49,22 +49,8 @@ if [[ $LOCK_EXIST_STATUS -gt 0 ]]; then
 fi
 set -e
 
-
-echo "Deploying IAM stack to $DEPLOYMENT"
-npx kes cf deploy --kes-folder app --region us-east-1\
- --deployment $DEPLOYMENT --template node_modules/@cumulus/deployment/iam
-
-echo "Deploying DB stack to $DEPLOYMENT"
-npx kes cf deploy --kes-folder app --region us-east-1\
- --deployment $DEPLOYMENT --template node_modules/@cumulus/deployment/db
-
-echo "Deploying APP stack to $DEPLOYMENT"
-npx kes cf deploy --kes-folder app --region us-east-1\
- --deployment $DEPLOYMENT --template node_modules/@cumulus/deployment/app
-
-echo "Deploying S3AccessTest lambda to $DEPLOYMENT"
-./node_modules/.bin/kes lambda S3AccessTest deploy \
-  --kes-folder app \
-  --template node_modules/@cumulus/deployment/app \
-  --deployment "$DEPLOYMENT" \
-  --region us-west-2
+if [[ $DEPLOYMENT =~ '-tf' ]]; then
+  . ./bamboo/bootstrap-tf-deployment.sh
```

Comment:
I like this - it cleanly separates without the hassle of another bamboo task.  👍     It's *probably* obvious from the output, but it might be worth a quick header echo for debugging, just to note which path the logic took here, but that's not a large issue either way.